source/UberCore/Configuration.swift

```diff
@@ -191,34 +191,36 @@ private let callbackURIStringKey = "URIString"
     // MARK: Getters
     
     /**
-     Gets the callback URIString of this app. Defaults to the value stored in your Application's
+     Gets the callback URI of this app. Defaults to the value stored in your Application's
      plist if not set (UberCallbackURI)
      
      - returns: The string to use for the Callback URI
     */
-    @objc public func getCallbackURIString() -> String {
-        return getCallbackURIString(for: .general)
+    @objc public func getCallbackURI() -> URL {
+        return getCallbackURI(for: .general)
     }
     
     /**
-     Gets the callback URIString for the given CallbackURIType. Defaults to the value 
+     Gets the callback URI for the given CallbackURIType. Defaults to the value
      stored in your Applications' plist (under the UberCallbackURIs key). If the requested
      type is not defined in your plist, it will attempt to use the .General type. If the 
      .General type is not defined, it will attempt to use the value stored under the UberCallbackURI key.
      Throws a fatal error if no value can be determined
      
      - parameter type: The CallbackURIType to get a callback string for
      
-     - returns: The callbackURIString for the the requested type
+     - returns: The callbackURI for the the requested type
      */
-    @objc public func getCallbackURIString(for type: CallbackURIType) -> String {
```

Comment:
Same here, can we keep an old deprecated version to remove in another couple versions.

Reply:
Specifically at the 1.0 cut?

Reply:
I am going to leave both method signatures in because of my comment here: https://github.com/uber/rides-ios-sdk/pull/197#discussion_r151299423